Makefile

```diff
@@ -338,3 +338,7 @@ ifndef DRONE_TOKEN
 	$(error DRONE_TOKEN is not set, visit https://drone.grafana.net/account)
 endif
 	DRONE_SERVER=https://drone.grafana.net drone sign --save grafana/tempo .drone/drone.yml
+
+
+# Import fragments
+include build/tools.mk
```

Comment:
I thought this was a good time to start breaking up the make config a bit, but if we like it all in one, I can move these targets into the `Makefile`.